docker/venv.Dockerfile

```diff
@@ -16,5 +17,7 @@ RUN if [ -n "${DEV}" ]; \
         pipenv install --deploy --system; \
     fi
 
+RUN [ -z "${NO_LINUXFS}" ] && pipenv run bootstrap-linux
```

Comment:
Avoid using `pipenv run` inside Docker because it will create a venv. It cannot know that a `--system` install was done.

Reply:
Any suggestions for this?

Reply:
Invoke the command directly in the Dockerfile. What is the rationale for having it in the Pipfile in the first place?

Reply:
I originally had the linuxfs on the host filesystem - having it in the Pipfile meant I could rapidly redeploy just the linuxfs without having to touch anything else. You're right though, I've reworked the logic and there's no reason for it to be there anymore. I'll move it straight into the Dockerfile.